packages/dbos-cloud/applications/deploy-app-code.ts

```diff
@@ -146,7 +146,11 @@ export async function deployAppCode(
     } else {
       logger.info("Time travel is disabled for this application");
     }
-    await registerApp(userDBName, host, enableTimeTravel, appName);
+    const ret = await registerApp(userDBName, host, enableTimeTravel, appName);
+    if (ret !== 0) {
+      logger.error(`Failed to deploy application ${appName}. Registration failed.`);
```

Comment:
Probably don't need this logging statement, because it appears to be two errors.